annet/annlib/tabparser.py

```diff
@@ -502,7 +503,7 @@ def blocks_and_context(
             else:
                 for row, _, row_context in row_group:
                     if context and context.parent and context.parent.row:
-                        prev_prow = context.parent.row
+                        prev_prow, prev_prow_context = context.parent.current
```

Comment:
Can `context.parent.current` be empty here? There's only a check for non-empty `context.parent.row`

Reply:
`parent.row` [checks](https://github.com/annetutil/annet/blob/ec5341e47ddb34bc7a84ff47fc6cb92426a5d10e/annet/annlib/tabparser.py#L58) `current` under the hood.